pip_review/__main__.py

```diff
@@ -58,8 +56,8 @@ def parse_args():
         '--auto', '-a', action='store_true', default=False,
         help='Automatically install every update found')
     parser.add_argument(
-        '--editables', '-e', action='store_true', default=False,
-        help='Also include editable packages in PyPI lookup')
+        '--user', '-u', action='store_true', default=False,
+        help='Only output packages installed in user-site.')
```

Comment:
Given that you are keeping `--user`, why are you removing `--editables`? At least in recent versions, `pip list` has an `--editable` option. Renaming would be an option, but see my next comment.